pkg/cluster/bootstrap.go

```diff
@@ -75,7 +74,7 @@ func (s *APIBootstrapper) Bootstrap(ctx context.Context, out io.Writer) error {
 			switch {
 			case errors.Is(err, context.DeadlineExceeded):
 				return retry.ExpectedError(err)
-			case status.Code(err) == codes.FailedPrecondition || status.Code(err) == codes.DeadlineExceeded:
+			case client.StatusCode(err) == codes.FailedPrecondition || client.StatusCode(err) == codes.DeadlineExceeded:
```

Comment:
While we are on it, I think it is worth adding comments about why we check error this way, why we check error string below, etc